app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -55,3 +55,13 @@
 //
 @include motion-ui-transitions;
 @include motion-ui-animations;
+
+
+$primary-color: null;
```

Comment:
This might help to remove some foundation deprecation warnings. It does not solve the problem that the underlying Gem is not activly maintained anymore